plugins/scaffolder-node/src/actions/util.ts

```diff
@@ -85,6 +85,10 @@ export const parseRepoUrl = (
       checkRequiredParams(parsed, 'project', 'repo');
       break;
     }
+    case 'azure': {
```

Comment:
Could have been a fall-through, but a case for azure seems to be the pattern here